### DISCUSSION TITLE
¿Apuntar al twitter corporativo o al técnico si lo hay?

### DISCUSSION THREAD

**Discussion by adrm** [PucelaBits/websegura#101] [Ideas]
Veo en este twit que la institución contestó diciendo que se hable directamente con el departamento técnico. En estos casos es mejor presionar a la institución en general o apuntar directamente al departamento técnico que son quienes tienen la capacidad de solucionarlo? Imagino que en muchos casos los técnicos son conscientes del tema pero tienen otras prioridades o limitaciones impuestas desde arriba.

https://twitter.com/lionel_ict/status/1359410318934499331

**Comment by nukeador** [comment]
Estoy seguro que los departamentos técnicos tienen la posibilidad de solucionarlo, pero quien toma las decisiones no son ellos. Se les puede avisar también si así nos lo dicen, pero creo que la presión en redes debe caer sobre el responsable de la web, que puede avisar fácilmente a la gente con la que trabajan.
